_studio/shared/umc/codec/h265_dec/src/umc_h265_va_supplier.cpp

```diff
@@ -174,7 +174,7 @@ UMC::Status VATaskSupplier::AllocateFrameData(H265DecoderFrame * pFrame, mfxSize
 H265Slice * VATaskSupplier::DecodeSliceHeader(UMC::MediaDataEx *nalUnit)
 {
     size_t dataSize = nalUnit->GetDataSize();
-    nalUnit->SetDataSize(std::min<size_t>(2560, dataSize));
+    nalUnit->SetDataSize(std::min<size_t>(2860, dataSize));
```

Comment:
why this is 2860, not 2861 or 2863? Can you, please, create a macro def or constant with the self-speaking name explaining the value. If 2860 is composite value - explain how it was constructed.

Reply:
Thanks Dmitry V. Pls review agian.

BTW,  will this comment be public to open source? Or just internal comment?
If public, is there any format/rule to comply will?

Reply:
This is open source project and this should be a public comment. Basically, not explaining what this magic value actually means and/or where it comes from undermines open source status of the project.